test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -1538,6 +1494,55 @@ public void testGetModifiedDocIds() throws Exception {
   }
 
   @Test
+  public void testGetModifiedDocIds_gsaAction() throws Exception {
+    executeUpdate("create table data(url varchar, action varchar, "
```

Comment:
This is fine, but I think it would be nice to pick on a different mode of operation, since we have a second test of the action column to play with here.